js/src/viewer.js

```diff
@@ -250,8 +280,9 @@
       if (!slotAddress) {
         return;
       }
+
       var windowConfig = {
-        manifest: this.manifests[options.loadedManifest],
+        manifest: this.state.getStateProperty('manifests')[options.loadedManifest],
         currentFocus : options.viewType,
         focusesOriginal : options.availableViews,
         currentCanvasID : options.canvasID,
```

Comment:
Here is an example where I did not refactor.  All of this information is coming from the initial configuration for any individual window objects.  We wrote Window so it sets all of the defaults on initialize, and overwrites those defaults by extending them with this configuration.  That means, we have to pass certain parameters, such as ```annotationCreationAvailable```, from Window to ImageView to Hud.  Ideally, Hud would get this information from the global state, but that means either 1) moving the default values out of Window and into SaveController, or 2) updating the SaveController with all of these values once Window has initialized (right now we only publish the value to SaveController if it changes from the default).